docs/getting_started/webviz-ert.rst

```diff
@@ -63,6 +51,8 @@ be turned off / on by clicking on the corresponding realization id on the right
 When multiple ensembles are selected each plot will be overlayed on top of
 each other transparently, where each ensemble gets its own color.
 
+![plot_viewer](https://user-images.githubusercontent.com/4508053/186850936-38a13f16-f795-4691-8455-fc12f8372d8e.gif)
```

Comment:
Think we should host these images as part of our documentation and check them into the repository. I don't know what guarantees Github has that these will be available in the future :thinking: